spec/modules/label_extractors/galaxy_zoo/euclid_spec.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe LabelExtractors::GalaxyZoo::Euclid do
+  let(:data_label_schema) do
+    {
```

Comment:
can we reference and use `LabelExtractors::GalaxyZoo::Euclid::TASK_KEY_LABEL_PREFIXES` and `TASK_KEY_DATA_LABELS` instead?

Reply:
Yes, done this